### ISSUE TITLE
MegaMekLab 0.43.4 Combat Vehicle Internal Structure Calculation

### ISSUE THREAD

**Issue by snrdg102417** [MegaMek/megameklab#127]
In BT TechManual PDF CAT 35002 2007-2013 The Topps Company, Inc. Corrected Third Printing Published by Catalyst Game Lab p. 99 calculates the Hephaestus Hovercraft OmniVehicle design example as having 3 structure points. MegaMekLab 0.43.4 calculates the internal structure points as being 2.

"He also determines that the Hephaestus will receive 3 internal structure points per location (10 percent of 30 tons = 0.10 x 30 tons = 3)."

**Comment by HammerGS** [comment]
I'm not sure where you are seeing this.
![image](https://user-images.githubusercontent.com/13752452/34332487-4514d1be-e8ee-11e7-8507-c817d93574cf.png)

and

![image](https://user-images.githubusercontent.com/13752452/34332491-5747899e-e8ee-11e7-9bc7-43e6868d0bec.png)

**Comment by snrdg102417** [comment]
My apologies for not being clear. On the Structure and Armor tab the internal structure weight of 3 tons is correct. On the Preview table of my copy of MegaMekLab downloaded 11/22/17 has the Internal Structure as 2.

Another apology for not providing a copy of the readout my attempts to including them have failed.

From what you have provided my guess is the copy of the application I'm using is out of date.  Now I cannot find the page to download the application, I think I need to go to sleep.

Thank you for the reply.

**Comment by snrdg102417** [comment]
I really need to get some sleep. Sorry about that.

**Comment by HammerGS** [comment]
Most current releases are always available from here - https://www.megamek.org/downloads

**Comment by snrdg102417** [comment]
Hello HammerGS,

Thank you for the link to the site which I've added to my favorites.  I stumbled on to the release page on Github and downloaded 0.43.6.

Using either MegMekLab 0.43.4 or 0.43.6  with the BT TechManual Hephaestus example I have gotten to the Allocate Tonnage For Internal Structure . I left the date at 3145 and changed to 3065 the Structure/Armor tab agrees that Internal Structure weighs three ton. Checking the preview tab both versions show list front, left, right, rear, and turret Internal Structure as two. 

Opening the file for the Hephaestus from Cache and checking the preview page shows Internal structure as 3.

I'm still not smart enough to include screen shoots or much of anything else so hopefully the verbage I've written is helpful.
